lib/diagnostic.js

```diff
@@ -6,33 +6,57 @@
 // splits it into an array of normalized words
 // (uppercase strings without punctuation)
 // and returns that array.
-const normalizeWords = function (/* */) {
-  /* your response here */
+const getNormalizedWords = function (text) {
+  let words = text.split(' ');
+  let normalizedWords = [];
+  for (let i = 0; i < words.length; i++) {
+    let word = words[i].toUpperCase().replace(/\W+/, '');
+    normalizedWords[i] = word;
+  }
+  return normalizedWords;
 };
 
 // 2. Write a function that takes a string argument
 // and returns an array of unique normalized words.
-const uniqueWords = function (/* */) {
-  /* your response here */
+const uniqueWords = function (text) {
+  let uniqueWords = {};
```

Comment:
You generally don't want to declare variables with the same name as its container.